src/query/storages/fuse/src/operations/read/runtime_filter_prunner.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use common_catalog::plan::PartInfoPtr;
+use common_exception::Result;
+use common_expression::ConstantFolder;
+use common_expression::Expr;
+use common_expression::FunctionContext;
+use common_expression::Scalar;
+use common_expression::TableSchema;
+use common_functions::BUILTIN_FUNCTIONS;
+use log::info;
+use storages_common_index::statistics_to_domain;
+
+use crate::FusePartInfo;
+
+pub fn runtime_filter_pruner(
+    table_schema: Arc<TableSchema>,
+    part: &PartInfoPtr,
+    filters: &Vec<Expr<String>>,
+    func_ctx: &FunctionContext,
+) -> Result<bool> {
```

Comment:
Do we need adding the runtime filter stats to explain? Now the stats:
```
├── pruning stats: [segments: <range pruning: 1 to 1>, blocks: <range pruning: 755 to 755, bloom pruning: 0 to 0>]
```

Reply:
current stats are collected before the pipeline runs, but runtime filter stats will be generated during the pipeline running. Maybe we can try to add runtime filter stats to query profile later.